packages/nc-gui/components/project/spreadsheet/components/editColumn/DateOptions.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-autocomplete
+    v-model="colMeta.date_format"
+    label="Date Format"
+    class="caption nc-column-name-input"
+    :rules="[isValidDateFormat]"
+    :items="dateFormatList"
+    dense
+    outlined
+  />
+</template>
+<script>
+import { dateFormat, validateDateFormat } from '~/helpers/dateFormatHelper'
+export default {
+  name: 'DateOptions',
+  props: ['column', 'meta', 'value'],
```

Comment:
can we add props type here?

Reply:
Hi @dammy001 sure, what would be the use of this prop?